NumbersLibrary/RationalNumber.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+
+namespace NumbersLibrary
+{
+    //Дробные числа или Рациональные числа
+    public class RationalNumber
+    {
+        private int _Numerator;//числитель
+        private int _Denumerator;//знаменатель
+
+        public int Numerator { get { return _Numerator; }private set { _Numerator = value; } }
+        public int Denumerator { get { return _Denumerator; } private set { _Denumerator = value; } }
```

Comment:
```suggestion
        private readonly int _Numerator;//числитель
        private readonly int _Denumerator;//знаменатель

        public int Numerator { get { return _Numerator; } }
        public int Denumerator { get { return _Denumerator; }  }
```
Дроби, вектора, комплексные числа - должны быть неизменяемыми.